R/showToast.R

```diff
@@ -9,6 +9,7 @@
 #' @param type length 1 character vector.  Valid values are "success", "error", "warning", and "info"
 #' @param message the toast message
 #' @param title the toast title.  Defaults to \code{NULL}
+#' @param keepVisible a logical indicating whether to the notification should remain visible until removed with \code{\link{hideToast}}. \code{FALSE} by default.
```

Comment:
Please update the docs for this para to "a logical.  If \code{TRUE}, the toast notification will remain visible until removed with \code{\link{hideToast}}.  If  \code{FALSE}, the default, the toast will automatically hide once the "showDuration" option has elapsed."